articles/azure-functions/functions-bindings-event-grid.md

```diff
@@ -347,7 +347,7 @@ For more information about how to create a subscription, see [the blob storage q
 You can get the system key by using the following API (HTTP GET):
 
 ```
-http://{functionappname}.azurewebsites.net/admin/host/systemkeys/eventgridextensionconfig_extension?code={masterkey}
+http://{functionappname}.azurewebsites.net/admin/host/systemkeys/eventgrid_extension?code={masterkey}
```

Comment:
Hi Marium, 

this is only when using version V2 of the Function Runtime. For V1 the old url still apply.